### PR TITLE
In Progress - Adding Documentation

### LINKED ISSUE #15: Documentation, Set up and Integration info, Screenshots missing
I think we are all aware what information has to go into a project documentation. Please include all necessary info and step by step info how to set it up for chat gateways (slack, facebook, gitter).

### PR DESCRIPTION
**Pull request by shivenmian**
@sudheesh001 @mariobehling Adding documentation for AskSusi Messengers. Fixes #15.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,38 @@
...
+
+![alt text](http://i.imgur.com/WXnSiGl.png "Susi Telegram")
+
+## Contributions
+
+We are currently adding more bots. Please help us expand Susi on other Social Media platforms. A list of platforms we aim to integrate Susi on can be seen [here](https://github.com/fossasia/asksusi_messengers/issues/1). You can propose and make Susi bots on other platforms too as you like.
+
+As of now, all the bots have been developed in ```node.js``` and ```Express``` for smooth builds and CI. If you wish to add more bots, please append your code in the ```index.js``` file (without altering the other bots), update the ```package.json``` with your ```npm``` dependencies, add the screenshots and the usage / deployment documentation in the ```README.md```, and send a **single** squashed PR containing all these changes.
+
+Thanks!
```
Comment (sudheesh001) [inline README.md]: Please add a new line after this line

Comment (shivenmian) [inline README.md]: @sudheesh001 done.

**Comment by mariobehling** [comment]
Thanks for getting this started. Please now add technical information and step by step set up instructions.

**Comment by shivenmian** [comment]
@mariobehling There is nothing to set up for the Facebook Messenger and Telegram bots, because they are on specified links and user just has to go to that link and chat, so they are standalone and can't really be integrated / set up anywhere.

For Slack, yes, you can set up your bot on any channel. That issue is covered in #16, I'm working on it.

**Comment by mariobehling** [comment]
> There is nothing to set up for the Facebook Messenger and Telegram bots, because they are on specified links 

How can I set up my own bot?

**Comment by shivenmian** [comment]
@mariobehling It doesn't make sense perhaps, does it? Because the Facebook bot is on a page named "Ask Susi", which is owned by us. The telegram bot is also owned by us and is at a specified location. So I don't think they can be used by anyone else.

Slack, yes, you can take my bot and deploy it to any channel you like.

@sudheesh001 What do you feel?

**Comment by mariobehling** [comment]
@shivenmian How did you set it up in your own location and how can I now do the same and set it up in my own location?

**Comment by mariobehling** [comment]
Sorry, I do not wanna be impolite but I am starting to doubt about my English. Apparently you set it up. So, since the beginning I am trying to find out how you set it up and ask for documentation about it. You did it - somehow, but how? I can now do my own research and find out, but we have a project here and that should be documented in the project.

I am just feeling I am asking the same question in different ways over and over again. After asking this question in different issues, chats and discussions, how should we close this topic? Maybe I should just let go.

**Comment by shivenmian** [comment]
@mariobehling 

No, I understand what you are saying. What you are saying is that since I set up on Messenger, Telegram and Slack, how can someone else set it up by themselves? I guess that is your question.

The answer I'm giving is that yes, the Slack Bot you can set up and I am working on that functionality. But for Messenger and Telegram, perhaps that question is not defined, because on Facebook and Telegram, bots don't work the same way as in Slack. 

In Slack, you have the feature of installing any bot in your channel and use it, right?

The point about FB and Telegram is that the bots are standalone. They aren't usually 'used' by someone else at a different location. Take the example of the Messenger bot at https://www.facebook.com/asksusisu. This is a Facebook page, right? When you message this page, the bot activates. You're saying you want to set up your own bot like that, but you see, the bot is at a defined location (an FB page by the name "Ask Susi"), so probably it doesn't make sense that you'll make a copy of it yourself. Same with Telegram.

I don't think bots on Facebook and Telegram work the same way as in Slack. In Slack, you can just invite the bot into your channel and you can talk to it. You can deploy the bot anywhere. On Facebook and Telegram, the bots are at a predefined location, and you need to go to that URL to chat with the bot, you don't make replicas of it. 

That is only my opinion. I may be wrong. I will let @sudheesh001 give his opinion, and we can then work accordingly.

**Comment by mariobehling** [comment]
> probably it doesn't make sense that you'll make a copy of it yourself. Same with Telegram.

This is an open source project. I would like to set it up myself. I dont care that it is in another location. I do not care if it makes sense to anyone. I would really like to do it. I wonder, if it would be possible to tell me, how this can be done.

**Comment by mariobehling** [comment]
> Slack Bot you can set up and I am working on that

But yes, it would be a good start to get the documentation for that. Not sure, why we are talking about this so much as documentation should be part of the development progress.

**Pull request merged**